server/memphis_handlers_stations.go

```diff
@@ -109,6 +109,14 @@ func validateReplicas(replicas int) error {
 	return nil
 }
 
+func validateIdempotencyWindow(retentionValue int, idempotencyWindow int, retentionType string) error {
```

Comment:
last thing, please move the retention type to be the first parameter

Reply:
OK, but I would like to learn why it matters :)

Reply:
Sure, nothing functional it is more for clean code and reasonable ordering of the parameters